### PR TITLE
Add Home Assistant MQTT discovery.

### PR DESCRIPTION
**Pull request by kmalisiewicz**
Add iSpindel to Home assistant via MQTT integration.
It is possible only if MQTT has been configured in iSpindel.

### CODE REVIEW

File: pio/lib/Sender/Sender.cpp [hunk 1]
```diff
@@ -115,6 +115,35 @@ bool SenderClass::mqttConnect(const String &server, uint16_t port, const String
...
+    if (response)
+    {
+        _mqttClient.setBufferSize(512);
+        auto chipid = String(ESP.getChipId(), HEX);
+        String device = "\"dev\": { \"name\": \"" + name + "\",\"mdl\": \"ispindel\",\"sw\": \"" + FIRMWAREVERSION + "\",\"mf\": \"iSpindel\",\"ids\": [\"" + chipid + "\"]}";
+        String topic = "homeassistant/sensor/" + chipid;
+        _mqttClient.publish((topic + "_temperature/config").c_str(), ("{ \"uniq_id\": \"" + chipid + "_temp\", \"dev_cla\": \"temperature\", \"name\": \"Temperature\", \"unit_of_meas\": \"°" + unit + "\", \"val_tpl\": \"{{ value_json }}\", \"stat_t\": \"ispindel/" + name + "/temperature\"," + device + "}").c_str());
+        _mqttClient.loop();
+        _mqttClient.publish((topic + "_tilt/config").c_str(), ("{ \"uniq_id\": \"" + chipid + "_tilt\", \"name\": \"Tilt\", \"val_tpl\": \"{{ value_json }}\", \"stat_t\": \"ispindel/" + name + "/tilt\"," + device + "}").c_str());
+        _mqttClient.loop();
```
Comment (universam1) [inline pio/lib/Sender/Sender.cpp:131]: For the sake of performance can the loop not be called eventually once and publish all together?

Comment (kmalisiewicz) [reply pio/lib/Sender/Sender.cpp]: Done.

**Review by universam1 — CHANGES_REQUESTED** [review]
sorry for the delay @kmalisiewicz 
the changes look good, only the git branch needs a rebase. 
Could you rebase please, happy to merge

**Review by universam1 — APPROVED** [review]
👍🏻

**Pull request merged**